### DISCUSSION TITLE
Were you able to fix iServices?

### DISCUSSION THREAD

**Discussion by AlluraRed** [iwissemben/Hackintosh-Opencore-Acer-Nitro-5-AN515-55-51QY#7] [Issues] [answered]
When i log into imessage it kicks me out 2 seconds in and i enter a login loop

**Comment by iwissemben** [comment, marked as answer]
Hi @AlluraRed,

To fix iServices you have to generate your **OWN** SMBIOS info (reusing existing info leads to conflicts and apple ID ban), add it to the `PlatformInfo` in the config.plist as per [this part](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/coffee-lake-plus.html#platforminfo) of the Opencore Guide and reinstall macOS with the updated config.plist in your efi folder.

I didn't include the data to it - except the `SystemProductName` entry - in the repository as each user has to generate its own SMBIOS, else it may create conflicts between Apple IDs I guess. 

Regarding the `SystemProductName` entry I've set it to "MacBookPro16,1" as it is matching my config, but I "MacBookPro16,4" should work too. Feel free to test it.

Please try that and give me a feedback, it'll help me add a section for that in the Readme.

Don't forget to stare the repo to give it more visibility.
